api/specs.proto

```diff
@@ -293,6 +294,9 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+	
+	// Run a custom init inside the container, if null, use the daemon's configured settings
```

Comment:
Please follow the surrounding doc style. The comment should start with the name of the field `// Init ...`.

Also, please this field up before `tty`.

Reply:
Done